targets/targets.json

```diff
@@ -4696,6 +4696,63 @@
             "ARM_MUSCA_B1"
         ]
     },
+    "ARM_MUSCA_S1": {
+        "inherits": [
+            "PSA_V8_M"
+        ],
+        "default_toolchain": "ARMC6",
+        "features_add": [
+            "EXPERIMENTAL_API"
+        ],
+        "forced_reset_timeout": 20,
+        "release_versions": [
+            "5"
+        ],
+        "core": "Cortex-M33-NS",
+        "supported_toolchains": [
+            "ARMC6",
+            "GCC_ARM"
+        ],
+        "device_has_add": [
+            "INTERRUPTIN",
+            "I2C",
+            "I2CSLAVE",
+            "LPTICKER",
+            "SERIAL",
+            "SLEEP",
+            "USTICKER"
+        ],
+        "macros_add": [
+            "__STARTUP_CLEAR_BSS",
+            "MBED_FAULT_HANDLER_DISABLED",
+            "CMSIS_NVIC_VIRTUAL",
+            "LPTICKER_DELAY_TICKS=3",
+            "MBED_MPU_CUSTOM",
+            "BL2"
+        ],
+        "extra_labels_add": [
+            "ARM_SSG",
+            "MUSCA_S1",
+            "MUSCA_S1_NS"
+        ],
```

Comment:
Please add detect code for Musca S1:
```
        "detect_code": [
            "5009"
        ]
```
You should also send a PR to update the platform database for mbed-ls
https://github.com/ARMmbed/mbed-os-tools/blob/master/src/mbed_os_tools/detect/platform_database.py#L298

Reply:
It is now added at line 4751. Platform database is also updated.